<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <!-- Arguments -->
  <arg name="time_step"             default="0.004"/>
  <arg name="output"                default="screen"/>
  <arg name="publish_measurements"  default="true"/>
  <arg name="launch_prefix"         default="nice -n -20" />
  <!-- <arg name="launch_prefix"         default="gdb -ex run -args" /> -->
  
  <!-- <rosparam command="load" ns="smb_lowlevel_controller" file="$(find smb_lowlevel_controller)/config/robot_parameters.yaml"/>
  <rosparam command="load" ns="smb_lowlevel_controller" file="$(find smb_lowlevel_controller)/config/ros.yaml"/>  
  -->

  <!-- Launch the lowlevel controller -->
  <node pkg="smb_lowlevel_controller" type="smb_lowlevel_controller_node" name="smb_lowlevel_controller" output="$(arg output)" launch-prefix="$(arg launch_prefix)">
    <param name="time_step"                   value="$(arg time_step)"/>
    <param name="standalone"                  value="false"/>
    <param name="publish_measurements"        value="$(arg publish_measurements)"/>
    <param name="publishing_time_step"        value="$(arg time_step)"/>
    <param name="sync_name"                   value="smb_sync"/>
    <param name="command_smb"                   value="false"/> <!-- Don't send commands so you can use the other joystick -->
  </node>

</launch>
